CHANGELOG.md

```diff
@@ -36,6 +38,8 @@ and this project adheres to
   - [#3249](https://github.com/bpftrace/bpftrace/pull/3249)
 - Faster map access for keyless maps by using BPF_MAP_TYPE_ARRAY
   - [#3300](https://github.com/bpftrace/bpftrace/pull/3300)
+- Multi map keys are stored as tuples
+  - [#3390](https://github.com/bpftrace/bpftrace/pull/3390/)
```

Comment:
Is this something users can see? I'd not include it if it's just an implementation detail. The other changelog entry is definitely something users would care about though